src/index.mjs

```diff
@@ -243,16 +248,29 @@ export function getMinMax(range, margin = 0) {
   };
 }
 
+/**
+ * Scale the given points into the range [from, to].
+ * 
+ * Points closer to min will be scaled closer to 'to'.
+ * Points closer to max will be scaled closer to 'from'.
+ * 
+ * Example - scalePoints(5, 30, 8, 20, [8, 9, 10, 11, 15, 16, 20])
+ * = [30, 27.916666666666668, 25.833333333333332, 23.75, 15.416666666666666, 13.333333333333332, 5]
+ * 
+ * Note: It is not necesarry for input range to include
+ * min and max.
+ * 
+ * @param from {Number} Lower bound of output range
+ * @param to {Number} Upper bound of output range
+ * @param min {number} Minimum value of the input range
+ * @param max {Number} Maximum value of the input range
+ * @param range {Number[]} Points within the range [min, max] to be scaled
+ * @returns {Number[]} Array of scaled points
+ */
 export function scalePoints(from, to, min, max, range) {
-  const minAllowed = from;
-  const maxAllowed = to;
-
   // NOTE: For explaination see: https://stackoverflow.com/a/31687097/1263876
```

Comment:
This comment specifically references a Stackoverflow comment about the now-removed scaling function. If we want to go along with the newly proposed function, we should also remove the comment.

Reply:
We can keep [this](https://stackoverflow.com/a/5295202/11370119) answer in the comments. The current function is based on that answer but instead of scaling the min to `a` we scale it to `b` and the max to `a`. Essentially reversing the output of the function in that answer.

Reply:
ok 👍🏼